src/patterns/equality-information/index.md.njk

```diff
@@ -0,0 +1,166 @@
+
+If your service covers more than one of England, Wales, Scotland or Northern Ireland, you should accommodate these differences in your design. For example, by changing the categories shown depending on where the user is based. Where this isn’t possible, use the English categories.
+
+{{ example({group: "patterns", item: "equality-information", example: "religion", html: true, nunjucks: true, open: false}) }}
+
+## Asking about sex and gender identity
+
+Use this approach to ask about sex and gender identity.
+
+{{ example({group: "patterns", item: "equality-information", example: "sex-gender", html: true, nunjucks: true, open: false}) }}
+
+## Asking about sexual orientation
+
+Use this approach to ask about sexual orientation.
+
+{{ example({group: "patterns", item: "equality-information", example: "sexual-orientation", html: true, nunjucks: true, open: false}) }}
+
+## Validation and error messages
+
+If a user enters information that’s valid but incomplete, accept it. For example, they might just enter their year of birth without the day or month.
+
+Error messages should be styled like this -
+
+{{ example({group: "patterns", item: "equality-information", example: "error-ethnicity", html: true, nunjucks: true, open: false}) }}
+
+{{ example({group: "patterns", item: "equality-information", example: "error-date-of-birth", html: true, nunjucks: true, open: false}) }}
+
+## Research on this pattern
+
+The two-step approach to asking for ethnicity information is based on:
+
+- user research by the [Race Disparity Unit](https://www.gov.uk/government/organisations/race-disparity-unit)
+- [examples of ways to ask about ethnic groups](https://designnotes.blog.gov.uk/2019/01/29/researching-how-we-ask-users-about-their-ethnicity/) from a number of different government services
+
+### Questions we’d like to answer through research
+
+For one-off services, we’ve suggested placing the equality and diversity questions between the ‘check your answers’ and confirmation pages.
+
+This is based on where the ‘Do you want to create an account?’ question is placed in the ‘Get support if you’re clinically extremely vulnerable to coronavirus’ service - but we’d like to confirm whether this placement works for the equality questions too.
+
+We’d also appreciate any the following questions:
+
+- do users understand from the first ‘Equality questions’ screen that the questions they’re about to answer are different from other questions in the service?
+- do we need to explain why we’re asking each question individually, or is it enough to have an explanation at the start?
+- what’s the best way to ask for equality information in different types of service where you expect to have more than one contact with the user?
+- is including the `If you prefer not to say, continue without entering any information` text the clearest way to get across that all the text inputs on that screen are optional?
```

Comment:
Is `If you prefer not to say, continue without entering any information` intentionally in backticks? It's displaying as inline code.